### PR TITLE
fix misuse of hidelogwindow

### PR DESCRIPTION
**Pull request by cocoon**
fix black screen if hidelogwindow=yes
https://github.com/neutrinolabs/xrdp/issues/414

### CODE REVIEW

File: xrdp/xrdp_wm.c [hunk 1]
```diff
@@ -567,7 +567,10 @@ xrdp_wm_init(struct xrdp_wm *self)
     xrdp_wm_load_static_pointers(self);
     self->screen->bg_color = self->xrdp_config->cfg_globals.ls_top_window_bg_color;
 
-    if (self->session->client_info->rdp_autologin || self->hide_log_window)
+    /* fix black screen if hidelogwindow=yes in xrdp.ini, only the log window should be hidden. not everything */
+    /* https://github.com/neutrinolabs/xrdp/issues/414 */
+    /*if (self->session->client_info->rdp_autologin || self->hide_log_window)*/
```
Comment (metalefty) [inline xrdp/xrdp_wm.c]: The fix seems good but don't leave the old code in comment. You can just remove it.

Comment (cocoon) [reply xrdp/xrdp_wm.c]: should I remove all other comments or leave them in place?

Comment (metalefty) [reply xrdp/xrdp_wm.c]: I think no comments needed there.

**Comment by metalefty** [comment]
Can you squash commits into one commit and include "Fixes #414." like 06ce0d7? Then #414 will be automatically closed when this PR merged.

**Pull request merged**